nixos/modules/services/web-servers/apache-httpd/mediawiki.nix

```diff
@@ -91,12 +91,16 @@ let
     };
 
     skins = config.skins;
+    extensions = config.extensions;
 
     buildPhase =
       ''
         for skin in $skins; do
           cp -prvd $skin/* skins/
         done
+        for extension in $extensions; do
+          cp -prvd $extension/* extensions/
```

Comment:
Would it be better to have each extension in it's own store path and using symlinks here?

Also you don't need the `/*` if you are using `-r`.